source/blood/src/triggers.cpp

```diff
@@ -1182,12 +1172,12 @@ void OperateSprite(int nSprite, XSPRITE *pXSprite, EVENT event)
     case kTrapFlame:
         switch (event.cmd) {
             case kCmdOff:
-                if (!SetSpriteState(nSprite, pXSprite, 0, event.causedBy)) break;
+                if (!SetSpriteState(nSprite, pXSprite, 0)) break;
                 seqSpawn(40, 3, pSprite->extra, -1);
                 sfxKill3DSound(pSprite, 0, -1);
                 break;
             case kCmdOn:
-                if (SetSpriteState(nSprite, pXSprite, 1, event.causedBy)) break;
+                if (!SetSpriteState(nSprite, pXSprite, 1)) break;
```

Comment:
If I set a breakpoint here, what do I need to do in the game to hit it?

Reply:
Nevermind, it is not important anymore.

Reply:
Just if somebody comes here, breakpoint can be triggered on E3M5 (Spare Parts) at the little room near the start with the scanner on the ceiling where 2 axe zombies will come in.